examples/mixture-mpi.py

```diff
@@ -55,8 +55,18 @@
 # from mirgecom.prometheus import UIUCMechanism
 import cantera
 import pyrometheus as pyro
+import sys
+if sys.version_info < (3, 9):
+    # importlib.resources either doesn't exist or lacks the files()
+    # function, so use the PyPI version:
+    import importlib_resources
+else:
+    # importlib.resources has files(), so use that:
+    import importlib.resources as importlib_resources
+
 
 logger = logging.getLogger(__name__)
+mechdata = importlib_resources.files("mechanisms")
```

Comment:
I think this is ultimately mis-scoped: It effectively introduces a top-level package `mechanisms`, when IMO `mirgecom.mechanisms` would be more appropriate.

x-ref: https://github.com/illinois-ceesd/mirgecom/pull/207/files

Reply:
I'm trying to figure out how to make this happen, but not having much luck with it.

Reply:
4742033  Ok, this is much closer to what we want here, but I think there's still some un-needed infrastructure stuff.